### PR TITLE
Add documentation about services for Mazda integration 

### PR DESCRIPTION
**Pull request by bdr99**
## Proposed change

Add documentation about newly added services for the Mazda integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/51016
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by emontnemery** [comment]
Please document input parameters for the services.

**Comment by bdr99** [comment]
@emontnemery Added documentation for input parameters.

File: source/_integrations/mazda.markdown [hunk 1]
```diff
@@ -55,6 +55,50 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
     The "Automatic Re-Lock" feature will automatically re-lock the doors if they are not opened shortly after being unlocked. This applies regardless of whether you are using the key, or unlocking the doors remotely using Home Assistant or the MyMazda app.
 </div>
 
+## Services
+
+This integration offers several services for interacting with Mazda vehicles. All of the services require the following input parameter:
+
+| Parameter | YAML ID | Description |
```
Comment (frenck) [inline source/_integrations/mazda.markdown]: Please take a look at other documented services on the format/title of the headers.

File: source/_integrations/mazda.markdown [hunk 2]
```diff
@@ -55,6 +55,50 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
+| --------- | ------- | ----------- |
+| Vehicle | `device_id` | The vehicle to execute the service for. In YAML mode, this is the device ID of the vehicle. |
+
+### Start engine
+
+Starts the vehicle engine. The vehicle engine can only be remotely started 2 consecutive times. To reset this counter, the vehicle must be driven.
```
Comment (frenck) [inline source/_integrations/mazda.markdown]: The service name is missing (applies to all services).

Please check other services documentation for a reference.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by bdr99** [comment]
@frenck OK, I updated it to follow the standard format that other integrations are using. I'm a little surprised that this standard format doesn't include the friendly names for the services and their parameters. I would think it would make more sense to document those, since the friendly names are what is displayed to the user. For example, "Start Engine" instead of `start_engine` since the former is what the user actually sees in the UI. They would only see `start_engine` if they switch to YAML mode.

File: source/_integrations/mazda.markdown [hunk 3]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
 
-This integration offers several services for interacting with Mazda vehicles. All of the services require the following input parameter:
+This integration offers several services for interacting with Mazda vehicles.
 
-| Parameter | YAML ID | Description |
-| --------- | ------- | ----------- |
-| Vehicle | `device_id` | The vehicle to execute the service for. In YAML mode, this is the device ID of the vehicle. |
-
-### Start engine
+### Service `start_engine`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.start_engine`
```

File: source/_integrations/mazda.markdown [hunk 4]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
 
 Starts the vehicle engine. The vehicle engine can only be remotely started 2 consecutive times. To reset this counter, the vehicle must be driven.
 
-### Stop engine
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `device_id` | yes | The device ID of the vehicle to start |
+
+### Service `stop_engine`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.stop_engine`
```

File: source/_integrations/mazda.markdown [hunk 5]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
 
 Stops the vehicle engine. This only works if the vehicle was remotely started.
 
-### Start charging
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `device_id` | yes | The device ID of the vehicle to stop |
+
+### Service `start_charging`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.start_charging`
```

File: source/_integrations/mazda.markdown [hunk 6]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
 
 Starts charging the vehicle battery. This only works with electric vehicles.
 
-### Stop charging
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `device_id` | yes | The device ID of the vehicle to start charging |
+
+### Service `stop_charging`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.stop_charging`
```

File: source/_integrations/mazda.markdown [hunk 7]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
 
 Stops charging the vehicle battery. This only works with electric vehicles.
 
-### Send POI
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `device_id` | yes | The device ID of the vehicle to stop charging |
 
-Send a GPS location to the vehicle's navigation system as a POI (Point of Interest). Requires a navigation SD card installed in the vehicle.
+### Service `send_poi`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.send_poi`
```

File: source/_integrations/mazda.markdown [hunk 8]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
-| POI name | `poi_name` | A friendly name for the location. |
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `device_id` | yes | The device ID of the vehicle to send the GPS location to |
+| `latitude` | yes | The latitude of the location to send. |
+| `longitude` | yes | The longitude of the location to send. |
+| `poi_name` | yes | A friendly name for the location. |
 
-### Turn on hazard lights
+### Service `turn_on_hazard_lights`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.turn_on_hazard_lights`
```

File: source/_integrations/mazda.markdown [hunk 9]
```diff
@@ -57,48 +57,67 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
...
 
 Turn on the vehicle hazard lights. The lights will flash briefly and then turn off.
 
-### Turn off hazard lights
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `device_id` | yes | The device ID of the vehicle to turn hazard lights on |
+
+### Service `turn_off_hazard_lights`
```
Comment (frenck) [inline source/_integrations/mazda.markdown]:
```suggestion
### Service `mazda.turn_off_hazard_lights`
```

**Comment by frenck** [comment]
> OK, I updated it to follow the standard format that other integrations are using. I'm a little surprised that this standard format doesn't include the friendly names for the services and their parameters. 

We agree. Recently the service description format in the core repository has been changed and currently is updated completely. This allows us to generate (a more user-friendly) and even more standardized approach for the near future. Right now, we stick to what we have.

**Comment by bdr99** [comment]
@frenck Added the "mazda" prefix like you suggested.
> 
> 
> > OK, I updated it to follow the standard format that other integrations are using. I'm a little surprised that this standard format doesn't include the friendly names for the services and their parameters.
> 
> We agree. Recently the service description format in the core repository has been changed and currently is updated completely. This allows us to generate (a more user-friendly) and even more standardized approach for the near future. Right now, we stick to what we have.

Great! Glad to hear that changes are in the works.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**